src/http/Response.js

```diff
@@ -71,4 +71,19 @@ export default class Response extends Message {
     prefix(messageWriter) {
         return messageWriter.responsePrefix(this);
     }
+
+    // returns the 'ID' field value of the response copied to from the request (if any)
```

Comment:
Let's try to simplify this definition:

```suggestion
    // The "corresponding Daft request" ID, as stored in response headers (or null).
    // Daft server transactions store the received Daft request ID when generating responses.
```